src/MultiSlider.js

```diff
@@ -47,20 +47,22 @@ var MultiSlider = React.createClass({
   },
 
   xForEvent: function (e) {
-    var node = this.getDOMNode();
-    var bound = node.getBoundingClientRect();
-    var clientX = e.clientX;
-    if (useTouches()) {
+   var node = this.getDOMNode();
+   var m = node.getScreenCTM();
+   var p = node.createSVGPoint(); 
+   p.x = e.clientX;
+   p = p.matrixTransform(m.inverse());
+   if (useTouches()) {
       // There is a bug in touch events and we need to compute the real clientX
       // http://stackoverflow.com/questions/5885808/includes-touch-events-clientx-y-scrolling-or-not
-      clientX = e.pageX; // safer to start with pageX
+      var clientX = e.pageX; // safer to start with pageX
       var stopAt = document.body.parentElement;
       while (node && node !== stopAt) {
         clientX -= node.scrollLeft;
-        node = node.parentElement;
+        p.x = clientX;
```

Comment:
why this change? it looks like this will break on mobile.
I think `xForEvent` function should not be changed.